KarhooSDK/Api/Request/Adyen/AdyenPaymentMethodsRequest.swift

```diff
@@ -9,17 +9,23 @@ import Foundation
 
 public struct AdyenPaymentMethodsRequest: Codable, KarhooCodableModel {
     
+    // The platform where a payment transaction takes place. This field can be used for filtering out payment methods that are only available on specific platforms. Possible values: * iOS * Android * Web
```

Comment:
Shouldn't be this doc added to reference?

Reply:
Same for shoperLocale

Reply:
I mean adding one more `/`

Reply:
According to the new styleguide there should indeed be 3 `/`. Fixed now in both comments :)